### PR TITLE
Added filters

### PR DESCRIPTION
**Pull request by MarcinGalewski**

### CODE REVIEW

File: src/Posts/PostDTO.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@ public class PostDTO
         public int? CommunityId { get; set; }
         public DateTimeOffset CreatedDateTime { get; set; }
         public int? ParentId { get; set; }
+        public bool IsDeleted { get; set; }
```
Comment (kubaa2002) [inline src/Posts/PostDTO.cs:15]: Czemu tu jest dodane IsDeleted? I tak serwer nie powinnien wysyłać usuniętych postów do fronta

File: src/Services/PostsService.cs [hunk 2]
```diff
@@ -12,28 +13,137 @@ namespace Projekt_Zespolowy.Services
...
+        List<Post> posts;
+        private readonly ICurrentUserService _currentUserService;
+        public PostsService(AppDbContext context, ICurrentUserService currentUserService)
+        {
+            this.context = context;
+            posts = new List<Post>();
+            for (int i = 0; i < 150; i++)
+            {
+                posts.Add( new Post() { Id = i , Content = $"test{i}", CommunityId = i%12 >= 8 ? null : i/12, AppUserId = (i%16).ToString(), ParentId = i%4 == 0 ? null : i/4, CreatedDateTime = DateTime.UtcNow});
+            }
```
Comment (kubaa2002) [inline src/Services/PostsService.cs:28]: lista postów była już usunięta w innym commicie czemu pojawia się tu znowu?

File: src/Services/CurrentUserService.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                                           .Select(u => u.Id)
+                                           .FirstOrDefault();
+
+                _cachedUserId = userIdFromDb;
+
+                return _cachedUserId;
+            }
+        }
+    }
+}
```
Comment (kubaa2002) [inline src/Services/CurrentUserService.cs:47]: Cała to usługa jest nie potrzebna. Używaj user managera jak w UserController

**Review by kubaa2002 — CHANGES_REQUESTED** [review]

File: src/Services/PostsService.cs (hunk 2 above)
Comment (MarcinGalewski) [reply src/Services/PostsService.cs]: Miałem jeszcze starszą wersje, jeszcze sprzed tej zmiany

File: src/Services/CurrentUserService.cs (hunk 3 above)
Comment (MarcinGalewski) [reply src/Services/CurrentUserService.cs:47]: Zastąpiłem tą usługe user managerem

**Comment by kubaa2002** [comment]
Usuń jeszcze teraz nie potrzebne pliki CurrentUserService i ICurrentUserService

**Review by kubaa2002 — APPROVED** [review]
